surah/src/androidMain/kotlin/com/islamversity/surah/view/SurahView.kt

```diff
@@ -118,6 +118,7 @@ class SurahView(
                 }
 
                 if (it is AyaUIModel) {
+                    it.toolbarVisible = state.settingsState.toolbarForAyaOption
```

Comment:
you should set this value on every `aya`
prefebly in the processor

Reply:
or a better approach would be to to pass the `state.settingsState.toolbarForAyaOption` to every `ayaView` 
as another parameter
this way we don't have to map the aya list again